script/Ciel/Day 11/11_01_CIEL11_2_1.txt

```diff
@@ -181,23 +191,29 @@ It simply means they've taken predation to its ultimate form.
 [sha:20a7910039597576f62f1b71b28e452884ec7475]{
 -- Page 107, Offset 29362.
 -- 人間は種として優れている反面、一個の生命としては弱すぎる。自分たち以外のあらゆるものを犠牲にしなければ生存できない弱さは、構造的な『悪』に他ならない。
-Though humans are an outstanding species, they're simultaneously far too weak on the individual level.
- To be so weak that you have to sacrifice everything but yourself in order to live is nothing short of evil at a structural level.
+Though humans are an outstanding species, as individuals they remain pitifully weak.
+ Anything so weak that it must sacrifice everything other than itself it to survive is, in no uncertain terms, fundamentally evil.
 }
 [sha:74bd11828286a02d40e80906f0ce90543beb5bb6]{
 -- Page 108, Offset 29363.
 -- 知性の有無の違い、生態としての違いはあれ、人間は何かを捕食する事で生きている生物だ。
-No matter the differences in intelligence or ecology, humans still prey on others in order to survive.
+Humans will prey on anything in order to survive,
+ regardless of its ecology,
+ regardless even of whether or not it possesses intelligence.
```

Comment:
I think this is more trying to say something along the lines of "sure, what you're eating might be ecologically and intellectually different from yourselves, but you still do eat them, you piece of shit",  which is why my original TL was phrased the way it was. Though, I do realize it is a bit unclear.

>Whatever ecological and intellectual differences there may be, you humans still have to prey on other life in order to survive.